src/actions/helpers/getPlayersHelper.js

```diff
@@ -0,0 +1,5 @@
+const getPlayersPromise = (roomId) => {
+  return;
```

Comment:
Work in progress?

Reply:
@duyoji 
No, this is just a mock function to import the function from the directory.
When merging, I will delete this function.